app/models/retry_job_task.rb

```diff
@@ -0,0 +1,26 @@
+class RetryJobTask < LaunchJobTask
```

Comment:
@hsong-rh I think we should call this TowingTask instead of retry since it is not a retry of the launch.

Reply:
Since its an internal name we are not exposing to anyone, I think a TowingTask makes it easier to understand the concept but you could also call it RecoveryTask